husky_components/input/input-type.js

```diff
@@ -54,29 +50,60 @@ define([
                 }
             },
 
-            typeInterface = {
-                setValue: function(data) {
+            typeSetter = {
+                default: function(data) {
                     this.$el.data({
                         value: data
                     }).trigger('data-changed');
                 },
 
+                time: function(data) {
+                    var dateTime = Globalize.parseDate(data, 'HH:mm:ss');
+
+                    typeSetter.default.call(this, Globalize.format(dateTime, this.options.timeFormat));
+                }
+            },
+
+            typeGetter = {
+                default: function() {
+                    return this.$el.find('input').val();
+                },
+                time: function() {
+                    var value = typeGetter.default.call(this),
+                        dateTime = Globalize.parseDate(value, this.options.timeFormat);
+
+                    return Globalize.format(dateTime, 'HH:mm:ss');
+                },
+                date: function() {
```

Comment:
thats because the component is not very good you have to get the value from inside the component and there is no public api for that ... set is no problem you can put it on the data of the element ...